### PR TITLE
Remove broker test route, update api spec.

### PR DESCRIPTION
**Pull request by ethandmd**
#28

### CODE REVIEW

File: api_v1.yml [hunk 1]
```diff
@@ -67,12 +67,169 @@ info:
               application/json:
                 schema:
                   $ref: components/schemas/SimOutput
+          '400':
+            description: Job ID not found
+          '500':
+            description Enable to read views for valid job ID from database.
```
Comment (cartervluck) [inline api_v1.yml:73]: missing colon

File: api_v1.yml [hunk 2]
```diff
@@ -125,13 +282,21 @@ info:
...
-      SimSave:
+      GameInput:
+        type: object
+        properties:
+          username:
+            type: String
+            format: utf-8
+          sim: SimInput
+
+      JoinINput:
```
Comment (cartervluck) [inline api_v1.yml:293]: This should probably be `JoinInput`

**Review by cartervluck — APPROVED** [review]
couple typos but otherwise LGTM

File: api_v1.yml (hunk 2 above)
Comment (ethandmd) [reply api_v1.yml:293]: nice, thanks for catching.

**Pull request merged**